air/src/aux_table/bitwise_pow2/tests.rs

```diff
@@ -55,6 +126,101 @@ fn test_bitwise_frame(operation: Selectors, a: u32, b: u32, cycle_row: usize) {
+    curr: &[Felt],
+    next: &[Felt],
+    operation1: Selectors,
+    operation2: Selectors,
+) -> EvaluationFrame<Felt> {
+    // Hardcoded values of `a`,`b` and `output` values of previous cycle row.
+    let prev_input_a = Felt::new(10);
+    let prev_input_b = Felt::new(9);
+    let prev_output = Felt::new(8);
+
+    let mut current_cycle_row = vec![Felt::ZERO; TRACE_WIDTH];
+    let mut next_cycle_row = vec![Felt::ZERO; TRACE_WIDTH];
+
+    let mut current_bitstate = vec![Felt::ZERO; BITWISE_TRACE_WIDTH];
+    let mut next_bitstate = vec![Felt::ZERO; BITWISE_TRACE_WIDTH];
+
+    // calculating the input values of current and next cycle row.
+    let input_curr_a = aggregate_input_bits(&curr[0..NUM_DECOMP_BITS], prev_input_a);
+    let input_curr_b =
+        aggregate_input_bits(&curr[NUM_DECOMP_BITS..NUM_DECOMP_BITS * 2], prev_input_b);
+    let input_next_a = aggregate_input_bits(&next[0..NUM_DECOMP_BITS], input_curr_a);
+    let input_next_b =
+        aggregate_input_bits(&next[NUM_DECOMP_BITS..NUM_DECOMP_BITS * 2], input_curr_b);
+
+    // calculating the ouptut values of current and next cycle row.
+    let output_agg_curr_op1 = aggregate_row_output(&curr, operation1, prev_output);
+    let output_agg_next_op1 = aggregate_row_output(&next, operation1, output_agg_curr_op1);
+
+    // First two columns would be selector.
+    // Assign operation2 and operation1 to the current and next cycle row respectively.
+    for idx in 0..NUM_SELECTORS {
+        current_bitstate[idx] = operation2[idx];
+        next_bitstate[idx] = operation1[idx];
+    }
+
+    current_bitstate[A_COL_IDX] = input_curr_a;
+    current_bitstate[B_COL_IDX] = input_curr_b;
+    next_bitstate[A_COL_IDX] = input_next_a;
+    next_bitstate[B_COL_IDX] = input_next_b;
+
+    current_bitstate
+        [BITWISE_DECOMPOSITION_OFFSET..BITWISE_DECOMPOSITION_OFFSET + BITWISE_DECOMPOSITION_SIZE]
+        .copy_from_slice(&curr);
+    next_bitstate
+        [BITWISE_DECOMPOSITION_OFFSET..BITWISE_DECOMPOSITION_OFFSET + BITWISE_DECOMPOSITION_SIZE]
+        .copy_from_slice(&next);
+
+    current_bitstate[OUTPUT_COL_IDX] = output_agg_curr_op1;
+    next_bitstate[OUTPUT_COL_IDX] = output_agg_next_op1;
+
+    current_cycle_row[BITWISE_TRACE_OFFSET..BITWISE_TRACE_OFFSET + BITWISE_TRACE_WIDTH]
+        .copy_from_slice(&current_bitstate);
+    next_cycle_row[BITWISE_TRACE_OFFSET..BITWISE_TRACE_OFFSET + BITWISE_TRACE_WIDTH]
+        .copy_from_slice(&next_bitstate);
+
+    let frame = EvaluationFrame::<Felt>::from_rows(current_cycle_row, next_cycle_row);
+
+    frame
+}
+
+/// Generates the expected and calculated constraint result on an input frame.
+fn get_bitwise_frame_result(frame: EvaluationFrame<Felt>, row: usize) -> [Felt; NUM_CONSTRAINTS] {
+    let cycle_row = get_test_periodic_values(row);
+    let mut result = [Felt::ZERO; NUM_CONSTRAINTS];
+    let _expected = result;
+
+    bitwise::enforce_constraints(&frame, &cycle_row, &mut result, Felt::ONE);
+
+    result
+}
```

Comment:
The reference to "expected" in the comment can be removed and the following line can be deleted now that expected is not returned:

```
let _expected = result;
```